doc/SONiC_202006_Release_Notes.md

```diff
@@ -0,0 +1,134 @@
+
+
+# Security Updates
+
+1. Kernal upgraded from 4.9.110-3deb9u6 (SONiC Release 201904) to 4.9.168-1+deb9u5 in this SONiC release. 
+   Change log: https://tracker.debian.org/media/packages/l/linux/changelog-4.9.168-1deb9u5
+2. Docker upgraded from 18.09.2\~3-0\~debian-stretch to 18.09.8\~3-0\~debian-stretch. 
+   Change log: https://docs.docker.com/engine/release-notes/#18098 
+
+# Feature List
+
+#### Build Improvements 
+This document provides functional design and specifications of BFD protocol as defined in RFC 5880, 5881, 5882 and 5883.BFD protocol defines a method of rapid detection of the failure of a forwarding path by checking that the next hop router is alive. The protocol will be able to detect the forwarding path failure in milliseconds depending on the actual configuration. 
+<br> Refer [HLD document]() and below mentioned PR's for more details. 
+<br> **Pull Requests** : [3292](https://github.com/Azure/sonic-buildimage/pull/3292), [4117](https://github.com/Azure/sonic-buildimage/pull/4117), [4425](https://github.com/Azure/sonic-buildimage/pull/4425) 
+
+#### Bulk API for route
+This document provides functional design and specifications of BFD protocol as defined in RFC 5880, 5881, 5882 and 5883.BFD protocol defines a method of rapid detection of the failure of a forwarding path by checking that the next hop router is alive. The protocol will be able to detect the forwarding path failure in milliseconds depending on the actual configuration. 
+<br> Refer [HLD document]() and below mentioned PR's for more details. 
+<br> **Pull Requests** : [1238](https://github.com/Azure/sonic-swss/pull/1238)  
+
+#### D-Bus to Host Communications 
+This document describes a means (framework) for an application executed inside a container to securely request the execution of an operation ("action") by the host OS.This framework is intended to be used by the SONiC management and telemetry containers, but can be extended for other application containers as well.
+<br> Refer [HLD document](https://github.com/Azure/SONiC/blob/master/doc/mgmt/Docker%20to%20Host%20communication.md) and below mentioned PR's for more details. 
+<br> **Pull Requests** : [4840](https://github.com/Azure/sonic-buildimage/pull/4840)
+
+#### Debian 10 upgrade, base image,driver 
+
+<br> Refer [HLD document]() and below mentioned PR's for more details. 
+<br> **Pull Requests** : [145](https://github.com/Azure/sonic-linux-kernel/pull/145), [4711](https://github.com/Azure/sonic-buildimage/pull/4711) 
+
+#### Dynamic port break
+Ports can be broken out to different speeds with various lanes in most HW today. However, on SONiC, the port breakout modes are hard-coded in the profiles and only loaded at initial time. In case we need to have a new port breakout mode, we would potentially need a new image or at least need to restart services which would impact the traffic of the box on irrelevant ports. The feature is to address the above issues.
```

Comment:
Suggest " However, on SONiC, before this release, the port breakout modes ...."

Reply:
Done.

Reply:
I guess you omitted the "before this release" part, but fair enough